h_transport_materials/property_database/eurofer_97/eurofer_97.py

```diff
@@ -0,0 +1,183 @@
+import h_transport_materials as htm
+from h_transport_materials.property import Diffusivity, Solubility, Permeability
+from pathlib import Path
+
+import numpy as np
+
+u = htm.ureg
+
+aiello_permeability_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/aiello_2002/permeability.csv",
+    delimiter=",",
+)
+
+aiello_permeability_data_H = aiello_permeability_data[2:, :2]
+
+aiello_permeability_H = Permeability(
+    data_T=1000 / aiello_permeability_data_H[:, 0] * u.K,
+    data_y=aiello_permeability_data_H[:, 1]
+    * u.mol
+    * u.m**-1
+    * u.Pa**-0.5
+    * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="H",
+)
+
+aiello_permeability_data_D = aiello_permeability_data[2:, 2:]
+
+aiello_permeability_D = Permeability(
+    data_T=1000 / aiello_permeability_data_D[:, 0] * u.K,
+    data_y=aiello_permeability_data_D[:, 1]
+    * u.mol
+    * u.m**-1
+    * u.Pa**-0.5
+    * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="D",
+)
+
+aiello_diffusivity_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/aiello_2002/permeability.csv",
+    delimiter=",",
+)
+
+aiello_diffusivity_data_H = aiello_diffusivity_data[2:, :2]
+
+aiello_diffusivity_H = Diffusivity(
+    data_T=1000 / aiello_diffusivity_data_H[:, 0] * u.K,
+    data_y=aiello_diffusivity_data_H[:, 1] * u.m**2 * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="H",
+)
+
+aiello_diffusivity_data_D = aiello_diffusivity_data[2:, 2:]
+
+aiello_diffusivity_D = Diffusivity(
+    data_T=1000 / aiello_diffusivity_data_D[:, 0] * u.K,
+    data_y=aiello_diffusivity_data_D[:, 1] * u.m**2 * u.s**-1,
+    source="aiello_hydrogen_2002",
+    isotope="D",
+)
+
+chen_permeability_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/chen_2021/permeability.csv",
+    delimiter=",",
+)
+
+chen_permeability = Permeability(
+    data_T=1000 / chen_permeability_data[:, 0] * u.K,
+    data_y=chen_permeability_data[:, 1] * u.mol * u.m**-1 * u.MPa**-0.5 * u.s**-1,
+    source="chen_deuterium_2021",
+    isotope="D",
+)
+
+chen_diffusivity_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/chen_2021/diffusivity.csv",
+    delimiter=",",
+)
+
+chen_diffusivity = Diffusivity(
+    data_T=1000 / chen_diffusivity_data[:, 0] * u.K,
+    data_y=chen_diffusivity_data[:, 1] * u.m**2 * u.s**-1,
+    source="chen_deuterium_2021",
+    isotope="D",
+)
+
+chen_solubility = Solubility(
+    S_0=4.0e2 * u.mol * u.m**-3 * u.MPa**-0.5,
+    E_S=29.2 * u.kJ * u.mol**-1,
+    range=(623 * u.K, 823 * u.K),
+    note="Given in paper chen_deuterium_2021 bbut no experimental data points",
```

Comment:
```suggestion
    note="Given in paper chen_deuterium_2021 but no experimental data points",
```

Reply:
Also let's avoid writing chen_deuterium_2021, this is only useful when the bib file is there. Here we can just say "in the paper"